app/move/controllers/unassign.js

```diff
@@ -21,6 +22,20 @@ class UnassignController extends FormWizardController {
     next()
   }
 
+  checkEligibility(req, res, next) {
+    const { allocation, id, profile = {}, status } = req.move
+
+    if (['requested', 'booked'].includes(status)) {
+      return next()
+    }
+
+    res.render('move/views/unassign-ineligible', {
+      allocation,
+      moveId: id,
+      fullname: profile.person?.fullname,
```

Comment:
We have person directly on the move. Might it be better to get it from there?

Reply:
...erm, should that actually be the case? I thought the whole point of moving to a "profile" was to remove the association of "person" from a move?!

Reply:
The API docs also suggest that's not the case:

<kbd>![image](https://user-images.githubusercontent.com/3327997/96583964-6febf200-12d5-11eb-84c4-9a6a4b5233a7.png)</kbd>

Or are you saying that's something that we are doing on the frontend in the service transformer/middleware?

Reply:
It [looks like](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/18dc3637ae5a9ba6cd72874975b76ce2505c3559/common/services/move.js#L62) we're assuming the API returns this relationship in the service, but [the model suggests](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/18dc3637ae5a9ba6cd72874975b76ce2505c3559/common/lib/api-client/models.js#L3-L55) this relationship also doesn't exist.

I wonder if this is related to us transitioning from `V1` to `V2` of the API and whether we should remove this?

Reply:
I can't remember for sure whether we added the person to the move in the transform for backwards compatibility or as I seem to remember as a convenience... We can remove. But I'd like that to wait until after we've done all these overhauls we have in motion…

Reply:
>We can remove. But I'd like that to wait until after we've done all these overhauls we have in motion…

Yep, totally agree this is a future fix and not to be bundled as part of this. Have created [an issue](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/903) to keep track of it for now...